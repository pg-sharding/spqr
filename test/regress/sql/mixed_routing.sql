DROP TABLE IF EXISTS xxmixed;
CREATE TABLE xxmixed(id int);
INSERT INTO xxmixed (id) VALUES(1);
INSERT INTO xxmixed (id) VALUES(10);
INSERT INTO xxmixed (id) VALUES(11);
INSERT INTO xxmixed (id) VALUES(20);

INSERT INTO xxmixed (id) VALUES(21);
INSERT INTO xxmixed (id) VALUES(22);
INSERT INTO xxmixed (id) VALUES(29);
INSERT INTO xxmixed (id) VALUES(30);

SELECT * FROM xxmixed;
SELECT * FROM xxmixed WHERE id <= 10;
SELECT * FROM xxmixed WHERE id <= 20;
SELECT * FROM xxmixed WHERE id <= 30;
