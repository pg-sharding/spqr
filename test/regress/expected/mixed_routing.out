psql:sql/mixed_routing.sql:1: Shard match: sh1,sh2
DROP TABLE
psql:sql/mixed_routing.sql:2: Shard match: sh1,sh2
CREATE TABLE
psql:sql/mixed_routing.sql:3: Shard match: sh1
INSERT 0 1
psql:sql/mixed_routing.sql:4: Shard match: sh1
INSERT 0 1
psql:sql/mixed_routing.sql:5: Shard match: sh1
INSERT 0 1
psql:sql/mixed_routing.sql:6: Shard match: sh1
INSERT 0 1
psql:sql/mixed_routing.sql:8: Shard match: sh2
INSERT 0 1
psql:sql/mixed_routing.sql:9: Shard match: sh2
INSERT 0 1
psql:sql/mixed_routing.sql:10: Shard match: sh2
INSERT 0 1
psql:sql/mixed_routing.sql:11: Shard match: sh2
INSERT 0 1
psql:sql/mixed_routing.sql:13: Shard match: sh1,sh2
 id 
----
  1
 10
 11
 20
 21
 22
 29
 30
(8 rows)

psql:sql/mixed_routing.sql:14: Shard match: sh1
 id 
----
  1
 10
(2 rows)

psql:sql/mixed_routing.sql:15: Shard match: sh1
 id 
----
  1
 10
 11
 20
(4 rows)

psql:sql/mixed_routing.sql:16: Shard match: sh2
 id 
----
 21
 22
 29
 30
(4 rows)

